run_pipeline.py

```
@@ -332,83 +337,92 @@ def run_single_document(
+                            document_kg=knowledge_graph,
+                            document_id=doc_name,
+                            cross_doc_detector=detector
+                        )
+
+                        # Save unified KG + per-document backup
+                        manager.save(unified_kg, document_id=doc_name)
+
+                        # Get statistics
+                        doc_stats = manager.get_document_statistics(unified_kg)
+
+                        print_success(f"KG merge complete with cross-document relationships!")
```

Ruff (0.14.7)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)